core-aam/index.html

```diff
@@ -5711,6 +5711,48 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property"
+                    >Interface: <code>IAccessibleAction</code>, using the accessible nodes matching `aria-actions` IDREFs, if the referenced objects are in the accessibility tree</span
```

Comment:
Using IAccessibleAction is great. It means we don't need to make any binary changes. However, I doubt that existing ATs will honor this and do anything useful. Therefore, I have a timing and adoption concern. We should work to provide examples as soon as possible behind a flag for ATs to work with, so that updates get to users sooner than 3 years from a year from now. As it stands this also blocks <option> from having inner buttons in the new customizable select.